telegram_notifier.py

```diff
@@ -0,0 +1,125 @@
+import requests
+import os
+from dotenv import load_dotenv
+from app.models.payloads import *
```

Comment:
Medium severity: Using wildcard import from app.models.payloads reduces code clarity and may cause namespace collisions. Since only TelegramOutputSending is used, explicitly import just that to improve maintainability and reduce accidental import of unused symbols. Example:
```suggestion
from app.models.payloads import TelegramOutputSending
``` (Severity: medium, score: 5/10)